File: Algorithm/BubbleSort.m
% √∞≈›≈≈–Ú£¨‘≠÷∑
function Array=BubbleSort(Array)
len=length(Array);

for i=1:len-1
    for j=len:-1:i+1
        if Array(j)<Array(j-1)
            temp=Array(j-1);
            Array(j-1)=Array(j);
            Array(j)=temp;
        end
    end
end
end

File: Algorithm/Sort & Order Statistic/C/BubbleSort.c
void BubbleSort(int* A,int arraysize)
{
	int temp;
	for (int i = 0; i < arraysize - 1; i++)
		for (int j = arraysize - 1; j > i; j--)
			if (*(A + j) < *(A + j - 1))
			{
				temp = *(A + j);
				*(A + j) = *(A + j - 1);
				*(A + j - 1) = temp;
			}
}